src/components/FormFields/SequenceOrderField/SequenceTransfer.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from 'prop-types'
+import { Field, Transfer } from '@dhis2/ui'
+import SequenceTransferTitle from './SequenceTransferTitle'
+import CustomOption from './CustomOption'
+
+const { bool, arrayOf, shape, func, array, string } = PropTypes
+
+const SequenceTransfer = ({ options, input, meta }) => {
+    const { onChange, value } = input
+    const hasError = meta.touched && !!meta.error
+    const errorMessage = hasError ? meta.error : ''
+
+    return (
+        <Field
+            label={i18n.t('Job sequence')}
+            validationText={errorMessage}
+            error={hasError}
+            required
+        >
+            <Transfer
```

Comment:
Hmm maybe that's because I'm using a custom option. Could be that they require a little extra work to support that. I'll look into it.

Reply:
I've added the doubleclick handler. I noticed that in our native transfer options we don't use the onDoubleClick prop, but instead a custom solution: https://github.com/dhis2/ui/blob/master/components/transfer/src/transfer-option.js. We should probably swap that out. Also, I think it's a slightly confusing pattern personally, but since we already employ the pattern elsewhere it makes sense to at least have it here as well.

Reply:
Yeah as an established pattern for our apps it would be weird to not have it here.

Reply:
Created an issue for fixing this in the transfer: https://dhis2.atlassian.net/browse/LIBS-515